### PR TITLE
Fix amounts over 1 million cutting off

### LINKED ISSUE #2510: [Bug]: Amount displayed being cut-off when more than 5 digit values are placed.
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### Is this related to GoCardless, Simplefin or another bank-sync provider?

- [X] I have checked my server logs and could not see any errors there
- [ ] I will be attaching my server logs to this issue
- [ ] I will be attaching my client-side (browser) logs to this issue
- [X] I understand that this issue will be automatically closed if insufficient information is provided

### What happened?

I created a demo to show what I mean.
Below is a screenshot of the accounts page:
image
As you can see from the screenshot the values are being cut-off when it's the 'millions'. Any value greater than 000,000 is being cut-off.
Hiding decimal places in settings still doesn't solve the issue (and it shouldn't anyway, since I would need the decimals).

### What error did you receive?

N/A

### Where are you hosting Actual?

Docker

### What browsers are you seeing the problem on?

Firefox, Chrome, Microsoft Edge

### Operating System

Windows 11

### PR DESCRIPTION
**Pull request by psybers**
Fixes #2510.

This supports values up to `9,999,999.99` before it cuts them off.  That should cover the vast majority of users.

### CODE REVIEW

**Comment by psybers** [comment]
image

**Comment by psybers** [comment]
Not sure what I am doing wrong with the VRT, but locally I get:

```
[@actual-app/web]:   2 failed
[@actual-app/web]:     [chromium] › mobile.test.js:28:7 › Mobile › loads the budget page with budgeted amounts ────────
[@actual-app/web]:     [chromium] › onboarding.test.js:34:7 › Onboarding › creates a new budget file by importing YNAB4 budget 
[@actual-app/web]:   1 flaky
[@actual-app/web]:     [chromium] › budget.test.js:27:7 › Budget › renders the summary information: available funds, overspent, budgeted and for next month 
[@actual-app/web]:   31 passed (2.0m)
```

But no more screenshots were updated.  Then if I run it again:

```
[@actual-app/web]:   6 failed
[@actual-app/web]:     [chromium] › accounts.test.js:61:9 › Accounts › Budgeted Accounts › creates a transfer from two existing transactions 
[@actual-app/web]:     [chromium] › budget.test.js:27:7 › Budget › renders the summary information: available funds, overspent, budgeted and for next month 
[@actual-app/web]:     [chromium] › mobile.test.js:28:7 › Mobile › loads the budget page with budgeted amounts ────────
[@actual-app/web]:     [chromium] › mobile.test.js:49:7 › Mobile › opens the accounts page and asserts on balances ────
[@actual-app/web]:     [chromium] › onboarding.test.js:34:7 › Onboarding › creates a new budget file by importing YNAB4 budget 
[@actual-app/web]:     [chromium] › onboarding.test.js:52:7 › Onboarding › creates a new budget file by importing nYNAB budget 
[@actual-app/web]:   28 passed (2.5m)
```

And then the check on GH fails:

```
[@actual-app/web]:   1 failed
[@actual-app/web]:     [chromium] › transactions.test.js:29:7 › Transactions › checks the page visuals ────────────────
[@actual-app/web]:   33 passed (1.4m)
```

File: packages/desktop-client/e2e/accounts.test.js-snapshots/Accounts-closes-an-account-1-chromium-linux.png
Comment (youngcw) [inline packages/desktop-client/e2e/accounts.test.js-snapshots/Accounts-closes-an-account-1-chromium-linux.png:1]: Something weird must have happened where you lost the new account in the sidebar.

**Review by youngcw — APPROVED** [review]

**Comment by WaterBloom** [comment]
Hi! I have this issue on a different page, under /reports/net-worth. (I'm not sure if I should open a new issue, didn't want to spam.)
image

**Comment by psybers** [comment]
> Hi! I have this issue on a different page, under /reports/net-worth. (I'm not sure if I should open a new issue, didn't want to spam. I use a currency significantly weaker than EUR/USD, meaning bigger numbers, so this will probably come up a lot.)

@WaterBloom Yes, open an issue.

**Comment by psybers** [comment]
@WaterBloom Maybe #2818 fixes your issue.

**Pull request merged**